relayer/relayMsgs.go

```diff
@@ -138,7 +138,8 @@ func (r *RelayMsgs) SendWithController(src, dst *Chain, useController bool) {
 
 		if r.IsMaxTx(msgLen, txSize) {
 			// Submit the transactions to src chain and update its status
-			r.Succeeded = r.Succeeded && send(src, msgs)
+			_, success, _ := src.SendMsgs(msgs)
```

Comment:
cc @michaelfig

Reply:
That LGTM (the next line preserves the semantics I need).